File: practical_5/armstrong.c
#include <stdio.h>

int main() {
    int num, original, remainder, sum = 0;

    printf("Enter a number: ");
    scanf("%d", &num);

    original = num;

    while (original != 0) {
        remainder = original % 10;
        sum += remainder * remainder * remainder;
        original /= 10;
    }

    if (sum == num)
        printf("%d is an armstrong number\n", num);
    else
        printf("%d is not an armstrong number\n", num);

    return 0;
}

File: practical_5/armstrong.py
num = int(input("Enter a number: "))
sum = 0

original = num

while original != 0:
   remainder = original % 10
   sum += remainder ** 3
   original //= 10

if num == sum:
   print(num, "is an armstrong number")
else:
   print(num, "is not an armstrong number")
